.travis.yml

```diff
@@ -48,6 +47,7 @@ geosx_linux_build: &geosx_linux_build
 
 geosx_osx_build: &geosx_osx_build
   stage: build
+  name: Mac_OSX
```

Comment:
I would remove those two lines to keep symmetry with `geosx_linux_build` which does not hold its name nor its stage.